lib/ansible/modules/cloud/packet/packet_device.py

```diff
@@ -114,7 +112,7 @@
 '''
 
 EXAMPLES = '''
-# All the examples assume that you have your Packet api token in env var PACKET_API_TOKEN.
+# All the examples assume that you have your Packet api token in env var c(PACKET_API_TOKEN).
```

Comment:
No need for `C(...)` here as the whole `EXAMPLES` block is already formatted correctly.